tests/sentry/flags/endpoints/test_logs.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime, timezone
+
+from django.urls import reverse
+
+from sentry.flags.models import FlagAuditLogModel
+from sentry.testutils.cases import APITestCase
+
+
+class OrganizationFlagLogIndexEndpointTestCase(APITestCase):
+    endpoint = "sentry-api-0-organization-flag-logs"
+
+    def setUp(self):
+        super().setUp()
+        self.login_as(user=self.user)
+        self.url = reverse(self.endpoint, args=(self.organization.id,))
+
+    @property
+    def features(self):
+        return {"organizations:feature-flag-ui": True}
+
+    def test_get(self):
+        model = FlagAuditLogModel(
+            action=0,
+            created_at=datetime.now(timezone.utc),
+            created_by="a@b.com",
+            created_by_type=0,
+            flag="hello",
+            organization_id=self.organization.id,
+            tags={"commit_sha": "123"},
+        )
+        model.save()
+
+        with self.feature(self.features):
+            response = self.client.get(self.url + "?statsPeriod=14d")
+            assert response.status_code == 200
+
+            result = response.json()
+            assert len(result["data"]) == 1
+            assert result["data"][0]["id"] == 1
+            assert result["data"][0]["action"] == "created"
+            assert "created_at" in result["data"][0]
+            assert result["data"][0]["created_by"] == "a@b.com"
+            assert result["data"][0]["created_by_type"] == "email"
+            assert result["data"][0]["flag"] == "hello"
+            assert result["data"][0]["tags"] == {"commit_sha": "123"}
```

Comment:
if it's not too much work, could we add a test that the statsperiod/start/end filtering works?